src/WarcraftLegacies.Source/Setup/Legends/LegendLordaeron.cs

```diff
@@ -80,7 +80,7 @@ public LegendLordaeron(PreplacedUnitSystem preplacedUnitSystem, ArtifactSetup ar
       CapitalPalace.AddProtector(preplacedUnitSystem.GetUnit(Constants.UNIT_H007_IMPROVED_CANNON_TOWER_LORDAERON_TOWER,
         new Point(9545, 9372)));
       CreateTrigger()
-        .RegisterUnitEvent(CapitalPalace.Unit, EVENT_UNIT_DEATH)
+        .RegisterUnitEvent(CapitalPalace.Unit, EVENT_UNIT_CHANGE_OWNER)
```

Comment:
As discussed, we need to check if Capital Palace is owned by someone other than Lordaeron.